tests/fugue_dask/test_sql.py

```diff
@@ -0,0 +1,22 @@
+import dask.dataframe as dd
+import pandas as pd
+from fugue import register_execution_engine
+from fugue_sql import FugueSQLWorkflow
+
+from fugue_dask import DaskExecutionEngine
+
+
+def test_sql():
+    register_execution_engine(
```

Comment:
so for now, i have not make them automatically registered.
Because to make the registration work, you still have to import fugue_spark, but IDE will complain you import it but you don't use it. That can be confusing. So I haven't figured out a nice way to automatically register spark and dask.